### PR TITLE
add body emphasized var

### PR DESCRIPTION
**Pull request by alysiawu**
add body emphasized var as font weight 600

### CODE REVIEW

**Comment by torreyatcitty** [comment]
Do you have a link to the updated design guide from Josiah were these have been defined?

I think the intention was to only update the styles for L4, where this is going to update for everything which is not what we want I don't think

**Comment by alysiawu** [comment]
Since I notice the heading font weight emphasized across all levels are the same weight, so I deduce for body should be the same... but checking w Josiah now..  ```$heading-font-weight--emphasized--L0: 600;
$heading-font-weight--emphasized--L1: 600;
$heading-font-weight--emphasized--L2: 600;
$heading-font-weight--emphasized--L3: 600;
$heading-font-weight--emphasized--L4: 600; ```

File: scss/base/_typography.scss [hunk 1]
```diff
@@ -266,6 +272,8 @@ body {
   --body-line-height: #{$body-line-height--L1};
   --body-paragraph: #{$body-paragraph--L1};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L1};
```
Comment (alysiawu) [inline scss/base/_typography.scss:275]: notice --header font weight was using the L1 variable, so using the L1 for body as well

**Review by torreyatcitty — APPROVED** [review]
Yup this makes sense to me. LGTM.

@mykelp would you like to also take a quick look over to confirm it matches the original intention?

File: scss/base/_typography.scss [hunk 2]
```diff
@@ -123,6 +123,12 @@ $body-paragraph--L2: 0.75rem;
 $body-paragraph--L3: 0.5rem;
 $body-paragraph--L4: 0.5rem;
 
+$body-font-weight--emphasized--L0: 500;
+$body-font-weight--emphasized--L1: 500;
+$body-font-weight--emphasized--L2: 500;
+$body-font-weight--emphasized--L3: 500;
+$body-font-weight--emphasized--L4: 500;
+
```
Comment (mykelp) [inline scss/base/_typography.scss:131]: This isn't quite correct. you need to add all the variables

```scss
$body-font-family--emphasized--L0: "Haas Grot Text R";
$body-font-family--emphasized--L1: "Haas Grot Text R";
$body-font-family--emphasized--L2: "Haas Grot Text R";
$body-font-family--emphasized--L3: "Haas Grot Text R";
$body-font-family--emphasized--L4: "Haas Grot Text R";

$body-font-size--emphasized--L0: 0rem;
$body-font-size--emphasized--L1: 1.25rem;
$body-font-size--emphasized--L2: 1.0625rem;
$body-font-size--emphasized--L3: 0.9375rem;
$body-font-size--emphasized--L4: 0.8125rem;

$body-font-weight--emphasized--L0: normal;
$body-font-weight--emphasized--L1: 500;
$body-font-weight--emphasized--L2: 500;
$body-font-weight--emphasized--L3: 500;
$body-font-weight--emphasized--L4: 500;

$body-line-height--emphasized--L0: 0rem;
$body-line-height--emphasized--L1: 1.75rem;
$body-line-height--emphasized--L2: 1.5rem;
$body-line-height--emphasized--L3: 1.25rem;
$body-line-height--emphasized--L4: 1.125rem;

$body-paragraph--emphasized--L0: 0rem;
$body-paragraph--emphasized--L1: 1rem;
$body-paragraph--emphasized--L2: 0.75rem;
$body-paragraph--emphasized--L3: 0.5rem;
$body-paragraph--emphasized--L4: 0.5rem;
```

Comment (mykelp) [reply scss/base/_typography.scss:131]: This was just copied, but we'd need to fill those out with the correct values for each level

File: scss/base/_typography.scss (hunk 1 above)
Comment (mykelp) [inline scss/base/_typography.scss:275]: same as above this needs to have the full list of variables set

```scss
  --body-font-family--emphasized: #{$body-font-family--emphasized--L1};
  --body-font-size--emphasized: #{$body-font-size--emphasized--L1};
  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L1};
  --body-line-height--emphasized: #{$body-line-height--emphasized--L1};
  --body-paragraph--emphasized: #{$body-paragraph--emphasized--L1};

```

File: scss/base/_typography.scss [hunk 3]
```diff
@@ -310,6 +318,8 @@ body {
   --body-line-height: #{$body-line-height--L0};
   --body-paragraph: #{$body-paragraph--L0};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L0};
```
Comment (mykelp) [inline scss/base/_typography.scss:321]: ditto: same as above

File: scss/base/_typography.scss [hunk 4]
```diff
@@ -354,6 +364,8 @@ body {
   --body-line-height: #{$body-line-height--L1};
   --body-paragraph: #{$body-paragraph--L1};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L1};
```
Comment (mykelp) [inline scss/base/_typography.scss:367]: same as above

**Review by mykelp — CHANGES_REQUESTED** [review]
This isn't quite correct. Left snippets on how to update

File: scss/base/_typography.scss [hunk 5]
```diff
@@ -398,6 +410,8 @@ body {
   --body-line-height: #{$body-line-height--L2};
   --body-paragraph: #{$body-paragraph--L2};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L2};
```
Comment (mykelp) [inline scss/base/_typography.scss:413]: same as above

File: scss/base/_typography.scss [hunk 6]
```diff
@@ -442,6 +456,8 @@ body {
   --body-line-height: #{$body-line-height--L3};
   --body-paragraph: #{$body-paragraph--L3};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L3};
```
Comment (mykelp) [inline scss/base/_typography.scss:459]: same as above

File: scss/base/_typography.scss [hunk 7]
```diff
@@ -486,6 +502,8 @@ body {
   --body-line-height: #{$body-line-height--L4};
   --body-paragraph: #{$body-paragraph--L4};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L4};
```
Comment (mykelp) [inline scss/base/_typography.scss:505]: same as above

File: scss/base/_typography.scss [hunk 8]
```diff
@@ -545,6 +563,10 @@ p,
   font-weight: var(--body-font-weight);
   line-height: var(--body-line-height);
 
+  &--emphasized {
+    font-weight: var(--body-font-weight--emphasized);
```
Comment (mykelp) [inline scss/base/_typography.scss:567]: this needs all the values set 

```scss
    font-family: var(--body-font-family--emphasized);
    font-size: var(--body-font-size--emphasized);
    font-weight: var(--body-font-weight--emphasized);
    line-height: var(--body-line-height--emphasized);
```

**Comment by mykelp** [comment]
Ok I realize the issue now...we should revert all these changes. There was a name change in the CDS from `body--link` to `body--emphasized`. So all that's required is to change the `--link` to `--emphasized`. Then check in webb3 for all occurrences to `body--link` and update to `body--emphasized`

**Comment by torreyatcitty** [comment]
That sucks. @alysiawu let's hold off on finishing this until trx-history is out. Originally it was fine to update `body--emphasized` across the board but updating `body--link` to become `body--emphasized` as @mykelp says touches more views across the app that I would like us to do in a separate build for design to verify all the cases to make sure is fine.

Also, we should confirm with design that this rename was indeed the case with Josiah because I also didn't think this was the case reading the discord messages. I think we should have better versioning of the CDS in Figma and better tickets being opened by design for things we need to change in the CDS where we don't have to be the ones guessing what exactly was changed from memory and looking at the whole CDS.

**Pull request closed without merging**